Changelog.md

```diff
@@ -1,11 +1,16 @@
 ## OpenStudio-HPXML v1.10.0
 
 __New Features__
-- Allows detailed modeling of electric vehicles (batteries and charging/discharging) using `Vehicles` as an alternative to the simple EV charging `PlugLoad`.
+- Electric vehicle enhancements:
+  - Allows detailed modeling of electric vehicles (batteries and charging/discharging) using `Vehicles` as an alternative to the simple EV charging `PlugLoad`.
+  - Adds EV driving unmet hours output.
 - Allows requesting timeseries EnergyPlus output meters (e.g., `--hourly "MainsWater:Facility"`), similar to requesting EnergyPlus output variables.
 - BuildResidentialScheduleFile measure:
-  - Adds stochastic schedule generation for electric vehicle charging.
+  - Adds stochastic schedule generation for electric vehicle charging (using `Vehicles`).
```

Comment:
This clarifies that simple EV charging `PlugLoad` won't use this generated schedule?

Reply:
Yes. See the table [here](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#detailed-schedule-inputs) and the 5th item in https://github.com/NREL/OpenStudio-HPXML/issues/1929.